packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -153,6 +153,10 @@
                   <include>commons-codec:commons-codec</include>
                   <include>commons-io:commons-io</include>
                   <include>org.openjdk.jol:jol-core</include>
+                  <include>org.apache.hudi:hudi-aws</include>
+                  <include>software.amazon.awssdk:*</include>
+                  <include>org.apache.httpcomponents:httpclient</include>
+                  <include>org.apache.httpcomponents:httpcore</include>
```

Comment:
We should avoid this.  If AWS ecosystem is used, `hudi-aws-bundle` should be used with `hudi-utilities-bundle`: https://hudi.apache.org/releases/release-0.12.2#bundle-updates.

Reply:
so we basically remove all of this from the utilities bundle, and then its mandatory that user will have to pass both utilities bundle and aws bundle?

Reply:
So we can remove this change here and user will pass bundle then.